runner.go

```diff
@@ -530,6 +530,32 @@ func getPlatforms(c cliContext) []string {
 	return expandStringSlice(platforms)
 }
 
+func getExecutionMethod(c cliContext) (string, error) {
+	var crowd string
+	if crowd = c.String("crowd"); crowd != "" {
+		fmt.Println("RF CLI Deprecation: --crowd is deprecated, use --execution-method instead")
+	}
```

Comment:
Maybe put this in a `getCrowd` method that always gives this error, so the deprecation code is not infecting this method?

Reply:
I thought about that, but the whole method will need to be rewritten anyways when `crowd` is removed.